src/OpenTelemetry.Extensions/Internal/ActivityEventAttachingLogProcessor.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-#if NET462_OR_GREATER || NETSTANDARD2_0 || NET5_0_OR_GREATER
+#if NET462_OR_GREATER || NETSTANDARD2_0 || NET6_0_OR_GREATER
```

Comment:
😕 When isn't this true?

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/2dcc721429779658f4b63ee140aad6361924fd79/src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj#L4

Reply:
we have net6.0 in debug build. (it was net5.0)

Reply:
Though I think the directive is always `true`. #619